packs/cisco_umbrella_dns.yml

```diff
@@ -0,0 +1,10 @@
+AnalysisType: pack
+PackID: PantherManaged.Cisco.Umbrella
+Description: Group of all Cisco Umbrella DNS detections
+PackDefinition:
+  IDs:
+    - CiscoUmbrella.DNS.Blocked
+    - CiscoUmbrella.DNS.FuzzyMatching
+    - CiscoUmbrella.DNS.Suspicious
+    # Globals used in these detections
```

Comment:
missing a global or copied comment?

Reply:
Copy pasta, left in if we ever add in detections that require globals